arch/src/aarch64/regs.rs

```diff
@@ -1,43 +0,0 @@
-// Copyright 2022 Arm Limited (or its affiliates). All rights reserved.
```

Comment:
Why this is moved to Hypervisor crate?

Reply:
I have explained in the commit message because of circular dependencies between arch and hypervisor crate.